matrix-meetings-bot/test/MeetingService.test.ts

```diff
@@ -1712,7 +1712,7 @@ describe('test relevant functionality of MeetingService', () => {
       ); // ignores 'whiteboard-with-custom-content.id'
     });
 
-    test('add 1 widget, then add another', async () => {
+    test('add 2 widgets, then add  2 other', async () => {
```

Comment:
I don't see much value in changing these tests with the current solution. We should still keep the change in the `test_default_events.json` but these tests can be reverted in my opinion.